lib/dashing/cli.rb

```diff
@@ -59,17 +59,11 @@ def install(gist_id)
     def start(*args)
       port_option = args.include?('-p') ? '' : ' -p 3030'
       args = args.join(' ')
-      command = "bundle exec thin -R config.ru start#{port_option} #{args}"
+      command = "bundle exec puma #{port_option} #{args} config.ru"
       command.prepend "export JOB_PATH=#{options[:job_path]}; " if options[:job_path]
       run_command(command)
     end
 
-    desc "stop", "Stops the thin server"
-    def stop
-      command = "bundle exec thin stop"
-      run_command(command)
-    end
```

Comment:
The puma change runs the server directly rather than spinning up a background process. Stopping the server means Ctrl-C.

Thank you for the input...I figured this would be a change that needed discussion. I will look into the equivalent background-process behavior for Puma.